schemas/external/repo/common.schema.json

```diff
@@ -39,12 +39,12 @@
           "examples": ["2004-10-23T12:00:00-06:00"],
           "description": "The server date and time when the resource was created in the repository, such as when an asset file is first uploaded or a directory is created by the server as the parent of a new asset. The Date Time property should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."
```

Comment:
I thought we had agreed to use RFC 3339 for XDM, since we didn't need some of the 8601 extensions.

Reply:
Maybe. This PR preserves the existing definition, and changing this definition is outside the scope of this change. I respectfully suggest that any further discussion on this point is deferred to a separate PR.